pages/installing-slimefun.md

```diff
@@ -0,0 +1,45 @@
+# Installing Slimefun
+
+## Prerequisites
+* A 1.14.4 Spigot Minecraft server.
```

Comment:
Slimefun isn't made for Spigot, it is a Bukkit Plugin.

And hard-coding this version onto this page is not a good idea, you'd have to update it everytime a new Minecraft Version comes out.
Just tell them to use a Slimefun & CS-CoreLib Version that is compatible with their Version of Minecraft.